### PR TITLE
fix setblock command typo

### PR DESCRIPTION
**Pull request by Himself65**
I think this can as the example and also test project

### CODE REVIEW

File: test/mineflayer.test.js [hunk 1]
```diff
@@ -243,11 +243,12 @@ squid.supportedVersions.forEach((supportedVersion, i) => {
         await waitMessage(bot, 'You do not have permission to use this command')
         serv.getPlayer('bot').op = true
       })
-      test('can use /setblock', async () => {
+      test('can use /setblock', async (done) => {
         await Promise.all([waitSpawnZone(bot, 2), onGround(bot)])
         bot.chat('/setblock 1 2 3 95 0')
         let [, newBlock] = await once(bot, 'blockUpdate:' + new Vec3(1, 2, 3), { array: true })
         expect(newBlock.type).toEqual(95)
+        done()
```
Comment (Himself65) [inline test/mineflayer.test.js]: I don't know why this part always throws an error which display it time out. I don't think this part have any problem before.

**Comment by Himself65** [comment]
@rom1504 maybe i fixed the bug on setblock？

**Comment by Himself65** [comment]
and you need squash these commits when merging

File: src/lib/plugins/blocks.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports.player = function (player, serv) {
   }
 
   player.sendBlock = (position, blockType, blockData) => // Call from player.setBlock unless you want "local" fake blocks
-    player.behavior('sendBlock', {
+    player.behavior('sendblock', {
```
Comment (rom1504) [inline src/lib/plugins/blocks.js]: Why did you change that, was it wrong ? Are the docs up to date on this ?

File: test/plugin.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+const settings = require('../config/default-settings')
+const path = require('path')
+const fs = require('fs')
+
+describe('test import third-party plugins', () => {
+  const pluginPath = path.resolve(__dirname, '../', 'src', 'plugins', 'example')
+  const filePath = path.resolve(__dirname, '../', 'src', 'plugins', 'example', 'index.js')
+  beforeAll(() => {
+    if (!fs.existsSync(pluginPath)) {
+      fs.mkdirSync(pluginPath)
```
Comment (rom1504) [inline test/plugin.test.js]: Why are you writing the file here if you also commit it ?

File: src/lib/plugins/blocks.js (hunk 2 above)
Comment (rom1504) [reply src/lib/plugins/blocks.js]: Ok then it should be updated the other way round.
The correct spelling is sendBlock, see https://github.com/PrismarineJS/flying-squid/blob/master/docs/API.md#sendblock

File: test/plugin.test.js (hunk 3 above)
Comment (Himself65) [reply test/plugin.test.js]: After thinking twice, I think this unit test part is unuseful, so I removed it

File: src/lib/plugins/blocks.js (hunk 2 above)
Comment (Himself65) [reply src/lib/plugins/blocks.js]: Fixed

Comment (demipixel) [reply src/lib/plugins/blocks.js]: Oh wait, the command "setblock" shouldn't be changed...

**Comment by rom1504** [comment]
the vanilla command is `setblock`, see https://minecraft.gamepedia.com/Commands/setblock
let's keep the same name
